packages/nx-plugin/generators.json

```diff
@@ -10,6 +10,11 @@
       "schema": "./src/generators/configure-storybook/schema.json",
       "description": "configure-storybook generator"
     },
+    "configure-playwright": {
```

Comment:
I'd recommend follow existing patterns to be explicit what it does 

following https://nx.dev/nx-api/react/generators/cypress-component-configuration -> `playwright-component-configuration`